vignettes/tech-dm-class.Rmd

```diff
@@ -98,7 +98,13 @@ This means that you can use this, for example, on a postgres database that you a
 Another way of creating a `dm` object is calling `new_dm()` on a list of `tbl` objects:
 
 ```{r}
-base_dm <- new_dm(list(trees = trees, mtcars = mtcars))
```

Comment:
To be consistent with the rest of the vignette, where we always use NYC flights datasets.